cinema/serializers.py

```diff
@@ -1 +1,72 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
```

Comment:
Maybe it is better to add all usable fields here, instead of `__all__`, then it will be more readable